src/posix/README.md

```diff
@@ -0,0 +1,84 @@
+OpenThread POSIX app
+====================
+
+OpenThread supports running its core on POSIX and transmits radio frames through a radio transceiver.
+
+Current most platforms in [examples/platforms](../../examples/platforms) supports transceiver mode.
+
+The figure below shows the architecture of running OpenThread in such mode.
+
+```
++-------------------------+
+|     MLE TMF UDP IP      |
+|  MeshForwarder 6LoWPAN  |
+| _ _ _ _ _ _ _ _ _ _ _ _ |      spinel         +------------------+
+|    OpenThread Core      | <---------------->  | OpenThread Radio |
++-------------------------+     UART|SPI        +------------------+
+         POSIX                                          Chip
+```
+
+Build
+-----
+
+```sh
+# build core for POSIX
+make -f src/posix/Makefile-posix
+# build transceiver, where xxxx is the platform name, such as cc2538, nrf52840 and so on
+make -f examples/Makefile-xxxx
+```
+
+Test
+----
+
+**NOTE** Assuming build system is 64bit Linux. You can play just like the normal OpenThread as described in the [command line document](../../src/cli/README.md).
+You can also do radio diagnostic using the command [diag](../../src/cli/README.md#diag).
```

Comment:
...can also **perform radio diagnostics**..."

Also, I would suggest linking to the full diagnostics README instead: https://github.com/openthread/openthread/blob/master/src/diag/README.md